app/api-requests/rpa-api/cph-check.js

```diff
@@ -24,15 +24,10 @@ const restrictedToCattlePigAndSheepLivestock = (cphNumber) => {
   return !between(cphNumber.slice(sliceNo), slaughterHousesOrPoultry.MIN, slaughterHousesOrPoultry.MAX)
 }
 
-const containAtLeastOneValidCph = (cphNumbers) => {
-  if (typeof cphNumbers === 'undefined' || !Array.isArray(cphNumbers)) {
-    return false
-  }
-  const isContainAtLeastOneValidCph = cphNumbers.some(
+const containAtLeastOneValidCph = (cphNumbers) => cphNumbers
```

Comment:
this is a lot neater, but is cphNumber definitely never undefined when passed in here? If not maybe we could have `cphNumbers?.some(...)`

Reply:
Roger